### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by bnb** [bnb/good-first-issue#1]
The module is currently lacking tests. This is something that is 100% needed but also something I'm entirely unfamiliar with. Would 💚 some help on this!

**Comment by bnb** [comment]
We're now using Jest!

Some needed tests:
- `good-first-issue <project> -o` exits appropriately
- `log` returns a properly formatted output without any `undefined` (the repo name is especially tricky!)
- `require('good-first-issue') operates and returns an object with the correct shape, as expected

**Comment by hemal7735** [comment]
@bnb can we add task lists?
I've few comments over the first task. I doubt if it can be tested.
I've raised a question on that repo: https://github.com/sindresorhus/opn/issues/109

**Comment by bnb** [comment]
@hemal7735 I believe it may be on our end rather than opn's end – I less care about checking that the issue actually opened in a browser and more care that when passing the -o flag, the CLI actually exits.

Context: For a while, we weren't exiting and the CLI would continue running without actually doing anything, forcing the user to CMD + C to get out of it.

**Comment by bnb** [comment]
@hemal7735 and yes, feel free to add tasks!

**Comment by hemal7735** [comment]
@bnb yup, checking the process - exit code is doable. Thanks for the clarification.

**Comment by bnb** [comment]
I _believe_ it was solved with https://github.com/bnb/good-first-issue/commit/3cfb8e67c1ab0f2d7801952cfffeb0a710447567 but I'd always like to maintain a check that it's exiting correctly 😬

**Comment by moita69** [comment]
I would like to contribute to this issue.

**Comment by alexanderbrown21** [comment]
anything happened so far with this issue? has this been resolved?

**Comment by bnb** [comment]
@iamtabrezkhan feel free - there may have been some things done but improvements/enhancements always welcome.

**Comment by carlosdeque-s** [comment]
Hi, Im new at github and my experience qith code is from the last 2 years with some months of innactivity in between.

Is there any task I could do or somethings I need to learn if I only have a small experience in Js, Ruby, Java, C++, Python?

**Comment by bnb** [comment]
@carlosdeque-s feel free to poke around and try things out. Use the project, if you have an idea about how it could be better create an issue and I'll share feedback on it. If it's an affirmative, go ahead and start working on a PR - I can at the very least give you pointers in the right direction.

**Comment by zootechdrum** [comment]
hey @bnb Is this still a on going project ? The project seems really interesting.

**Comment by bnb** [comment]
@zootechdrum I don't actively work on it at this point for no reason other than I have other things I focus on. I could work on it; also happy to work with others to enable them to get improvements and additions shipped and eventually become a maintainer.

**Comment by ka1994ka** [comment]
هاي

**Comment by yabhinavy** [comment]
!assign

**Comment by adityashibu** [comment]
Hey everyone, I was wondering to get started with open source contribution. I'm an amateur in Python, OCaml, Javascript, HTML and CSS. Can anyone please explain how I can start working on this??

**Comment by AravindKVishwanath** [comment]
Hii everyone, I want to start open-source contribution. My skills are React, react native and I can also build basic backend for the applications. How can I start and continue open-source contribution?

**Comment by mr-dibya01** [comment]
Can I work on this issue?

**Comment by sne-ha091** [comment]
Is this issue still open ?
